File: ProAgil.Domain/Evento.cs
using System;
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Evento
    {
        public int ID { get; set; }
        public string Local { get; set; }
        public DateTime DataEvento { get; set;}
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<Lote> lote { get; set; }
        public List<RedeSocial> RedesSociais { get; set; }
        public List<PalestranteEvento> PalestrantesEventos { get; set; }
        public string ImagemUrl { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Evento.ts
import { Lote } from './Lote';
import { RedeSocial } from './RedeSocial';
import { Palestrante } from './Palestrante';

export interface Evento {
    id: number;
    local: string;
    dataEvento: Date;
    tema: string;
    qtdPessoas: number;
    telefone: string;
    email: string;
    lote: Lote[];
    redesSociais: RedeSocial[];
    palestrantesEventos: Palestrante[];
    imagemUrl: string;
}
